data/org.freedesktop.impl.portal.RemoteDesktop.xml

```diff
@@ -350,6 +350,26 @@
       <arg type="a{sv}" name="options" direction="in"/>
       <arg type="u" name="slot" direction="in"/>
     </method>
+    <!--
+        ConnectToEIS:
+        @session_handle: Object path for the #org.freedesktop.portal.Session object
+        @app_id: App id of the application
+        @options: Vardict with optional further information
+        @fd: A file descriptor to an EIS implementation that can be passed to a
+        sender libei context
+
+        Request a connection to an EIS implementation.
+
+        This method is available in version 2 of this interface.
```

Comment:
Adding new methods require bumping the interface version, I think - so that apps can then decide whether or not to call this method solely based on the version number.

Reply:
The version was bumped by the clipboard remote desktop integration pull request.

Reply:
Ah my bad then.